tests/test_page_inputs.py

```diff
@@ -72,7 +74,7 @@ def test_http_defaults(cls, body_cls):
     http_body = body_cls(b"content")
 
     obj = cls("url", body=http_body)
-    assert obj.url == "url"
+    assert str(obj.url) == "url"
```

Comment:
The tests so far only checks for equality when casted to a string. We could add assertions for inequality between a `_Url` and `str` to ensure they won't be equal.

Reply:
A good point, done!